ui_framework/src/components/flex/_flex_grid.scss

```diff
@@ -0,0 +1,60 @@
+.kuiFlexGrid {
```

Comment:
This was applied. Your gutter sizes vars were unnecessarily halved, so I changed that one bit. (They still apply margins by halves).